style.css

```diff
@@ -682,3 +685,84 @@ small {
   font-size: 13px;
   font-family: 'Inter', Arial, Helvetica, sans-serif;
 }
+
+.popTitle {
+  font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  margin-left: 0;
+  margin-top: 0;
+}
+
+#project-live-source {
+  margin-left: -40px;
+  justify-content: space-between;
+}
+
+#project-live-source li button a {
+  height: 35px;
+  width: 110px;
+  background: #ff6b00;
+  color: aliceblue;
+  border: rgb(255, 255, 255);
+  font-weight: 700;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+}
+
+#popup {
+  max-width: 45vw;
+}
+
+.popup-close {
+  position: absolute;
+  right: 30px;
+  top: 35px;
+  border: none;
+  background: #fff;
+}
+
+#technologies {
+  margin-left: -40px;
+  column-gap: 10px;
+}
+
+#technologies li button {
+  padding: 5px;
+  border: 1px solid rgb(222, 220, 220);
+  background: white;
+}
+
+.popup-image {
+  width: 100%;
+  height: 220px;
+}
+
+.popup-description {
+  width: 100%;
+  text-align: justify;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 13px;
+  line-height: 19px;
+}
+
+.pop-icon {
+  margin-left: 3px;
+}
+
+.popup {
+  position: fixed;
+  top: 50%;
+  left: 27%;
+  transform: translate(-50%, -50%);
+  z-index: 9999;
+  background-color: #fff;
+  padding: 20px;
+  box-shadow: 1px 20px 40px 20px rgb(102, 90, 90);
+}
```

Comment:
- [ ] The width of the pop-up is now too wide (it's almost 100% of the width of its container). Like the previous reviewer suggested, Kindly make sure the pop has a width closer to the one on the Figma design to make them look identical. I think `width: 81%` looks great 😎 
- [ ] The margin between the buttons is now too wide as well. Please adjust it to look just like the Figma design 👍 
## Figma Design
![image](https://user-images.githubusercontent.com/97834160/218284038-883c49aa-d6cc-4c67-b557-42fa89aa7e19.png)

## Your Design
![image](https://user-images.githubusercontent.com/46839548/218286998-ca5e9dee-cffe-46ab-8267-ea28583f3d60.png)